apps/site/src/app/apply/sections/Form/ProfileInformation.tsx

```diff
@@ -44,7 +44,7 @@ export default function ProfileInformation() {
 			<Textfield
 				name="job-answer"
 				labelClass={`${styles.label} mt-7`}
-				labelText="If you could do any job, what would it be? (ex. Youtuber, Body Builder, etc.) (100 words)"
+				labelText="If you could have any job in the world, what would it be? (ex. Youtuber, Body Builder, etc.) (100 words)"
```

Comment:
Nitpick: T in YouTuber should be capitalized, bodybuilder is more commonly written as one word